src/rebar3_typer_prv.erl

```diff
@@ -10,11 +10,53 @@
               {rebar_state, command_parsed_args, 1}]).
 
 -define(PROVIDER, typer).
--define(OPTS,
-        [{boolean, $b, "boolean", {boolean, false}, "Boolean example"},
-         {string, $s, "rebar-config", {string, "some.txt"}, "String example"},
-         {atom, $a, "ignore", atom, "Atom example"},
-         {atom_none, $n, "atom-none", {atom, none}, "Atom example, with default"}]).
+-define(OPTS, %% TODO: consider adding shorthand version to some (or all) options, even if it doesn't exist on TypEr itself
+        %% list of {Name,  ShortOpt,  LongOpt,    ArgSpec,   HelpMsg}
+        [{recursive,
+          undefined,
+          "recursive",
+          {boolean, false},
+          "Search directories recursively for .erl files below them."},
+         {show,
+          undefined,
+          "show",
+          {boolean, false},
```

Comment:
If the default behavior of typer is to have this option in `true`, then it should be true for us as well.

Reply:
Good point, I overlooked it :)